scripts/migrations/postgres/20211206165802_post_slug.down.sql

```diff
@@ -0,0 +1,26 @@
+BEGIN;
+
+DROP TRIGGER trigger_post_unique_slug ON post;
+
+ALTER TABLE post
+    DROP COLUMN slug;
+
+DROP FUNCTION IF EXISTS unique_post_slug();
+
+DROP MATERIALIZED VIEW calculated_post;
```

Comment:
There is no CREATE or REPLACE MATERIALIZED VIEW.

Reply:
I don't remember if you must refresh it once before using

Reply:
CREATE MATERIALIZED VIEW defines a materialized view of a query. The query is executed and used to populate the view at the time the command is issued (unless WITH NO DATA is used